src/DotNetBoilerplate.Core/Projects/Project.cs

```diff
@@ -1,57 +1,60 @@
-﻿namespace DotNetBoilerplate.Core.Projects
+﻿using DotNetBoilerplate.Core.Organizations.Exceptions;
+
+namespace DotNetBoilerplate.Core.Projects
 {
     public class Project
     {
         private Project() { }
         public Guid Id { get; private set; }
         public string Name { get; private set; }
         public string Description { get; private set; }
-        public string Status { get; private set; }
+        public ProjectStatus Status { get; private set; }
         public Guid OrganizationId { get; private set; }
         public Guid CreatorId { get; private set; }
         public DateTimeOffset CreatedAt { get; private set; }
 
-     /*   public enum ProjectStatus
+        public enum ProjectStatus
         {
             Public,
             Private,
             Archived
-        } */
+        } 
 
         public void UpdateName(string name, bool nameIsUnique)
         {
             if (!nameIsUnique)
             {
-                throw new Exception("Invalid name");
+                throw new ProjectNameIsNotUniqueException();
             }
 
             Name = name;
         }
 
-        public void UpdateStatus(string status)
+        public void UpdateStatus(ProjectStatus status)
         {
-            Status = status;
+            if (status == ProjectStatus.Public) { Status = ProjectStatus.Public; }
+            else if (status == ProjectStatus.Private) { Status = ProjectStatus.Private; }
+            else if (status == ProjectStatus.Archived) { Status = ProjectStatus.Archived; }
         }
 
         public static Project Create(
             string name,
             string description,
-            string status,
             Guid organizationId,
             Guid creatorId,
             DateTimeOffset createdAt,
             bool nameIsUnique
         )
         {
             if (!nameIsUnique)
-                throw new Exception("Invalid name");
+                throw new ProjectNameIsNotUniqueException();
 
             return new Project
             {
                 Id = Guid.NewGuid(),
                 Name = name,
                 Description = description,
-                Status = status,
+                Status = ProjectStatus.Private,
```

Comment:
Tutaj fajnie - to co mówiłem, piszę tylko ten comment odnośnie wcześniejszego commenta bo wcześniej normalnie zrobiliście  Status = status a nie przez if'y więc nie ma potrzeby wyżej ich stosować